test.rb

```
@@ -0,0 +1,28 @@
+
+def log_activity(msg)
+  File.open('log.txt', 'w') do |f|
+    f.write(msg + "\n")
+  end
+end
+
+data = read_file()
+process_data(data)
+log_activity("Finished processing at " + Time.now.to_s)
+
+unused_var = 42
```

RuboCop (1.75.5)

[warning] Useless assignment to variable - `unused_var`.

(Lint/UselessAssignment)